openmicroscopy/openmicroscopy
# Comments cleanup

# What this PR does
Cleans up the tests created in https://github.com/openmicroscopy/openmicroscopy/pull/5159, https://github.com/openmicroscopy/openmicroscopy/pull/5248 and https://github.com/openmicroscopy/openmicroscopy/pull/5275


# Testing this PR
 - Make sure the integration tests on roles still pass. 
 - Read the diff here and make sure the comments and javadoc for single tests make sense.
 - Check the ``@see`` parts for the javadoc first 12 tests in ``...integration/LightAdminRolesTest.java`` which should bring you to the PowerPoints explanation of these tests as g.doc, for example https://github.com/pwalczysko/openmicroscopy/blob/comments-cleanup/components/tools/OmeroJava/test/integration/LightAdminRolesTest.java#L198



# Related reading
https://trello.com/c/ygDN23CC/41-new-role-integration-tests-petr
https://trello.com/c/fdiVZar7/38-add-to-javadoc-the-link-to-test-explanations



cc @joshmoore @jburel @mtbc
